setup.py

```diff
@@ -302,7 +302,11 @@ def run(self):
         'console_scripts': [
             'stablehlo-to-saved-model = torch_xla.tf_saved_model_integration:main'
         ],
-        'torch_xla.plugins': ['tpu = torch_xla._internal.tpu:TpuPlugin',],
+        'torch_xla.plugins': [
+            'tpu = torch_xla._internal.tpu:TpuPlugin',
+            'neuron = torch_xla._internal.neuron:NeuronPlugin',
```

Comment:
so for every plug in we still need to modify the pytorch/xla to make them discoverable?

Reply:
No, these are just defined in our `setup.py` because the main package contains the plugin implementations. Any package can add more entry points separately, e.g.

https://github.com/pytorch/xla/blob/de283c986f8514b538037b192f64be7fa744592e/plugins/cuda/pyproject.toml#L17-L18